csharp/Platform.Ranges/Range[T].cs

```diff
@@ -58,89 +70,144 @@ public Range(T minimum, T maximum)
         }
 
         /// <summary>
-        /// <para>Presents the Range in readable format.</para>
-        /// <para>Представляет диапазон в удобном для чтения формате.</para>
+        /// <para>Returns a string representation of the current <see cref="Range{T}"/> struct.</para>
+        /// <para>Возвращает строковое представление текущей структуры <see cref="Range{T}"/>.</para>
         /// </summary>
-        /// <returns><para>String representation of the Range.</para><para>Строковое представление диапазона.</para></returns>
+        /// <returns>
+        /// <para>A string representation of the current <see cref="Range{T}"/> struct.</para>
+        /// <para>Строковое представление текущей структуры <see cref="Range{T}"/>.</para>
+        /// </returns>
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         public override string ToString() => $"[{Minimum}..{Maximum}]";
 
         /// <summary>
-        /// <para>Determines if the provided value is inside the range.</para>
-        /// <para>Определяет, находится ли указанное значение внутри диапазона.</para>
+        /// <para>Determines whether the current <see cref="Range{T}"/> struct contains the <paramref name="value"/>.</para>
+        /// <para>Определяет, содержится ли <paramref name="value"/> в текущей структуре <see cref="Range{T}"/>.</para>
         /// </summary>
-        /// <param name="value"><para>The value to test.</para><para>Значение для проверки.</para></param>
-        /// <returns><para>True if the value is inside Range, else false.</para><para>True, если значение находится внутри диапазона, иначе false.</para></returns>
+        /// <param name="value">
+        /// <para>A value to test for inclusion in the <see cref="Range{T}"/> struct.</para>
+        /// <para>Значение для проверки его присутствия в структуре <see cref="Range{T}"/>.</para>
+        /// </param>
+        /// <returns>
+        /// <para>A <see cref="Boolean"/> value that determines whether the <see cref="Range{T}"/> struct contains the <paramref name="value"/>.</para>
+        /// <para>Значение типа <see cref="Boolean"/>, определяющее, содержится ли <paramref name="value"/> в структуре <see cref="Range{T}"/>.</para>
+        /// </returns>
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         public bool Contains(T value) => _comparer.Compare(Minimum, value) <= 0 && _comparer.Compare(Maximum, value) >= 0;
```

Comment:
```suggestion
        public bool Includes(T value) => _comparer.Compare(Minimum, value) <= 0 && _comparer.Compare(Maximum, value) >= 0;
```
Note: this a breaking change. We should also ensure to use include instead of contain for Ranges everywhere. This is because word include is used with Range much more frequently.